app/views/applications/show.html.erb

```diff
@@ -0,0 +1,64 @@
+<h1 id="hdr1-show"> Application Show Page! </h1>
+
+<h2> Application Status </h2>
+<p id="app-status"><%= @application.status %></p>
+
+<h2>Applicant Contact Info</h2>
+<p id="app-name"><%= @application.name %></p>
+<p id="app-address">
+  <%= @application.street %>, 
+  <%= @application.city %>, 
+  <%= @application.state %> 
+  <%= @application.zip %>
+</p>
+
+<h3 id="why-good-home">How can I provide a good home? </h3> 
+<p id="app-descr"><%= @application.descr %></p>
+
+<section id='show-pets-on-app'>
+  <br>
+  <h2 id="hdr1-pets"> My Pets </h2>
+  <% @application.pets.each do |pet| %>
+    <p id="pet-name"><%= pet.name %></p>
+  <% end %>
+</section> 
+
+<% if @application.status == "In progress" %>
+  <section> 
+    <br>
+    <h1>Add a Pet to this Application</h1>
+    <br>
+    <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |form| %>
+      <%= form.hidden_field :search_pet  %>
+      <%= form.label "Search for pet" %>
+      <%= form.text_field :pet_name%>
+      <%= form.submit "Search" %>
+    <% end %>
+  
+    <% @pets.each do |pet| %> 
+    <section id=<%= "show-#{pet.id}" %>>
+      <%= form_with url: "/pet_applications?app_id=#{@application.id}", method: :post, local: true do |form| %>
```

Comment:
Awesome RESTful route again! Another RESTful option if you want to avoid the query param would be:
"/applications/:application_id/pets" --> it still mentions both resources involved in the join but has a path parameter. Both this route and the one you used that references the joins table are great RESTful options!